src/core/utils/remote-feature-config.ts

```diff
@@ -64,16 +70,13 @@ export const isFeatureActive = (feature: RemoteFeature): boolean => {
         return true;
     }
 
-    if (
-        (feature === RemoteFeature.NEAR ||
-            feature === RemoteFeature.COSMOS ||
-            feature === RemoteFeature.CELO ||
-            feature === RemoteFeature.DEV_TOOLS) &&
-        featuresConfig
-    ) {
+    if (featuresConfig) {
         const values = featuresConfig[feature] || [];
         if (values.length > 0) {
-            return values.indexOf(DeviceInfo.getUniqueId()) >= 0;
+            return (
```

Comment:
you need to add one more check here, `Array.isArray(values)` `values.length` will fail if values is a number for example